.ci/install_runtime.sh

```diff
@@ -118,3 +118,17 @@ if [ "$KATA_EXPERIMENTAL_FEATURES" = true ]; then
 	feature="newstore"
 	sudo sed -i -e "s|^experimental.*$|experimental=[ \"$feature\" ]|" "${runtime_config_path}"
 fi
+
+# Enable virtiofs if VIRTIO_FS is set to true
+# currently we use nemu for virtiofs testing
+if [ "$VIRTIO_FS" = true ] && [ "$KATA_HYPERVISOR" = "nemu" ]; then
+	echo "Configure virtio-fs on kata-runtime config file"
+	sudo crudini --set "$runtime_config_path" hypervisor.qemu virtio_fs_daemon "\"/usr/local/bin/virtiofsd-${arch}\""
```

Comment:
After https://github.com/kata-containers/tests/pull/1595, maybe we can remove the hardcoding of the virtiofsd binary and use `${KATA_NEMU_DESTDIR}`